data/crac-io/crac-io-json/src/main/java/com/powsybl/openrao/data/craciojson/serializers/CracSerializer.java

```diff
@@ -79,11 +80,6 @@ private void serializeNetworkElements(Crac crac, JsonGenerator gen) throws IOExc
                 .forEach(networkElement -> networkElementsNamesPerId.put(networkElement.getId(), networkElement.getName()));
         }
 
-        // Get network elements from Contingencies
```

Comment:
Not sure this deletion is okay ?

Reply:
This comment is about removing contingency elements from Crac own elements list?
Yes it was discuss with @pet-mit, it is ok to remove.
I couldn't keep it as before because previous rao network elements have Name and core contingency elements have not, and so when I had same elements coming from contingency (id=1, no name) or from another Crac objects (id=1, name=name1) these created error because you should have same name (null vs name1) when same element (same id).